### DISCUSSION TITLE
sweethome3d 7.1 "Can't save preferences in filesystem"

### DISCUSSION THREAD

**Discussion by thenger** [ScoopInstaller/Scoop#5625] [Help and Questions]
Hi after a successful install, preferences cannot be saved.
(I am not admin on the post I'm testing it)
does someone encountered the same problem ?

### Error
`Can't save preferences in filesystem`

### version
```
> scoop info sweethome3d

Name        : sweethome3d
Description : A free interior design application that helps you draw the plan of your house, arrange furniture on it and visit the results in 3D
Version     : 7.1
Bucket      : extras
Website     : http://www.sweethome3d.com
License     : GPL-2.0-or-later
Updated at  : 2023-07-22 12:17:43
Updated by  : HUMORCE
Installed   : 7.1
Binaries    : SweetHome3D.exe
Shortcuts   : Sweet Home 3D
```

### Steps to reproduce:
1. `scoop install sweethome3d`
2. Open sh3 and draw something
3. Change a preference: `ctrl+m` then change units for exemple
4. Save file `ctrl+shift+s`
5. Close sh3
6. Reopen sh3
7. Load saved file
8. Message appears and settings are effectively lost

**Comment by rashil2000** [comment]
According to the manifest https://github.com/ScoopInstaller/Extras/blob/master/bucket/sweethome3d.json#L47, the files should persist. Can you check them manually after changing a setting?
